analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/Processor.cs

```diff
@@ -26,5 +27,10 @@ namespace SonarAnalyzer.SymbolicExecution.Roslyn.OperationProcessors;
 internal abstract class Processor<T>
     where T : IOperationWrapper
 {
+    /// <summary>
+    /// Stop processing this branch of the exploded graph. There will be no follow up states.
+    /// </summary>
+    protected static readonly ProgramState[] EmptyStates = Array.Empty<ProgramState>();
```

Comment:
I know it's a duplicate 😒  The purpose of it is to be handy.

Reply:
It also causes a code smell warning: https://sonarcloud.io/project/issues?pullRequest=6115&issues=AYN6P16kIUgTTJgfgxLD&open=AYN6P16kIUgTTJgfgxLD&id=sonaranalyzer-dotnet

You may find a better place for it (property on ProgramState itself?).

Reply:
It would be hard to find there (one would already had to know to look for it there). The purpose of this helper is to be handy and easy to find and use. It doesn't need to be the source of truth (singleton)